### DISCUSSION TITLE
Database for SSR

### DISCUSSION THREAD

**Discussion by svenhue** [CodeDredd/pinia-orm#5] [Q&A] [answered]
In the vuex orm they talk about fetching data from the database instance for ssr...
see -> https://vuex-orm.org/guide/model/retrieving-models.html#fetch-from-database

How does pinia orm handles this? Do i have to import my model in each vue component where i want to use it? 
I saw the Database class in the code of this repo but i doesnt now how to access it.

Thank you

**Comment by svenhue** [comment, marked as answer]
After looking at the source code i finally got it....

**Reply by CodeDredd** [reply]
@svenhue hey sven. First of all. THANKS FOR THE SUPPORT ❤️ ❤️‍🔥 
You are the first who supports my open source work. Going to add you 😉 

Good that you already found it. Documentation is going to be updated. Right now i am on vaccation till 20.4. So i think is going to be after that time.
